setup.py

```diff
@@ -23,7 +24,7 @@ def read(*filenames, **kwargs):
     sep = kwargs.get("sep", "\n")
     buf = []
     for fl in filenames:
-        with codecs.open(os.path.join(HERE, fl), "rb", encoding) as f:
+        with codecs.open(HERE / fl, "rb", encoding) as f:
```

Comment:
```suggestion
        with (HERE / fl).open("r", encoding) as f:
```
You should open the file in text mode, not binary mode and then apply the encoding. Path objects provide a `Path.open()` method.

Since when is the abbreviation of filename `fl`?